e2e/cypress/integration/hashtags_spec.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import '@testing-library/cypress/add-commands';
+
+/**
+ * Note : This test requires the demo plugin tar file under fixtures folder.
+ * Download :
+ * make dist latest master and copy to ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.9.0.tar.gz
+ */
+
+describe('Integrations', () => {
+
+    const pluginIdDemo = 'com.mattermost.demo-plugin'
+    const demoFile = 'com.mattermost.demo-plugin-0.9.0.tar.gz';
+
+    before(() => {
+        cy.apiLogin('sysadmin')
+        cy.visit('/');
+
+        cy.apiRemovePluginById(pluginIdDemo, "");
+
+        cy.apiUploadPlugin(demoFile);
+        cy.apiEnablePluginById(pluginIdDemo);
+    });
+
+    after(() => {
+        cy.apiRemovePluginById(pluginIdDemo, "");
+    });
+
+    it.only('MM-T3426 Hashtags still work with demo plugin enabled', () => {
+        // # Post a hashtag
+        cy.get('#post_textbox').clear().type('#pickles {enter}');
+
+        cy.getLastPostId().then((postId: string) => {
+
+          // # click hashtag in from the last post
+          cy.get(`#postMessageText_${postId}`).
+            find('.mention-link').
+            click();
+
+          // * verify post exists in the RHS
+          cy.get(`#searchResult_${postId}`).
+              should('contain.text', '#pickles')
+          });
```

Comment:
The spacing here (only 2 instead of 4) seemed to be not consistent with others.

Reply:
@saturninoabril, I see why this is happening. I have the `e2e` directory parallel to the `webapp` directory, there for it isn't getting the `.eslintrc.json` file.  

Do you think we would be better off moving e2e inside the webapp dir, or possible just linking `webapp/.eslintrc.json` into the `e2e` directory?

Reply:
Another issue I am seeing by having e2e outside of webapp..  `make check-style` does not pick up the e2e directory.  We can fix this in the Makefile by checking for the existence of the `e2e` dir, but I wonder if changing the Makefile seems correct.  Maybe we could discuss some of this in a zoom?

Reply:
Hello Jason, I just read this now but sure, let's talk in zoom. For the meantime, IMO, I agree and it makes sense to move the `/e2e` folder inside the `/webapp` folder to primarily align with the dependencies and tooling.

Reply:
@saturninoabril, done!  `e2e` moved into `webapp` so `make check-style` catches linting errors inside the `e2e` folder.  I matched some of the options from `mattermost-webapp` `.eslintrc.json`
- max nested was 3 which is not going to work for cypress because it is two statements already with the `describe` and `it` blocks.
- fix unresolved errors